app/models/asset.rb

```diff
@@ -184,6 +184,12 @@ def role
     end
   end
 
+  def self.destroy_deleted_accounts_older_than_30_days
+    Asset.destroyable.find_each do |asset|
+      Asset::DestroyWithRelations.new(asset: asset).call
```

Comment:
Kind of curious if we have other options with regards to syntax here. This is be a bit "chewy" for me.

Reply:
Would you elaborate? :)